Source/DafnyLanguageServer.Test/Lookup/HoverVerificationTest.cs

```diff
@@ -60,13 +60,13 @@ method Abs(x: int) returns (y: int)
       await AssertHoverMatches(documentItem, (5, 4),
         @"[**Error:**](???) A postcondition might not hold on this return path.  
 Could not prove: y >= 0  
-This is assertion #1 of 4 in method Abs  
+This is assertion #2 of 4 in method Abs  
 Resource usage: ??? RU"
       );
       await AssertHoverMatches(documentItem, (7, 11),
         @"[**Error:**](???) assertion might not hold  
-This is assertion #2 of 4 in method Abs  
-Resource usage: 9K RU"
+This is assertion #1 of 4 in method Abs  
+Resource usage: ??? RU"
```

Comment:
Do you know why we lost this?

Reply:
The `???` matches anything, so that change just makes the test less fragile as the solver evolves.